tox.ini

```diff
@@ -15,6 +15,7 @@ commands=
     doctest: make -C {toxinidir}/docs doctest
 deps=
     eth-hash[pycryptodome]
+    mypy
```

Comment:
Seems a little overkill to have to install mypy on all the tox environments. Maybe we could just run that type inference in the lint run, and remove it from all the py* tests. Then we can drop the special pypy3 section, and this line could be dropped.

Reply:
:+1: done